server/paths/apply.ts

```diff
@@ -24,6 +25,7 @@ const paths = {
     update: applicationPath,
     checkYourAnswers: applicationPath.path('check-your-answers'),
     submission: applicationPath.path('submission'),
+    withdrawables: { new: withdrawablesPath.path('new') },
```

Comment:
With my REST hat on, this feels a bit weird. We're not creating a withdrawable here, if anything, this feels like we're initialising a withdrawl. I think we should use the `WithdrawlsController#new` action instead.